_build/utils/pepper_runner.py

```diff
@@ -98,7 +98,7 @@ def run_pepper(pepper_params,full_log=False):
 	sys.__stdout__.write(" " *30 + "\n")
 
 	script_dir = os.path.dirname(os.path.realpath(__file__)) + os.sep
-	annis_out_dir = script_dir + ".." + os.sep + "target" + os.sep + "annis" + os.sep
+	annis_out_dir = script_dir + ".." + os.sep + ".." + os.sep + "annis" + os.sep
```

Comment:
This path should evaluate to `_build/target/annis/`, not the top level `annis/` directory, for two reasons: reddit files should not be generated in the pushable root directories, and we generally test the (possibly messy) output in target/ before carefully moving it (minus any reddit data) to the root level folders